test/basic_features/utils_test.py

```diff
@@ -18,19 +18,19 @@ def setUp(self):
   def test_options_get(self):
     self.assertEqual(requests.get(self.url_options).status_code, 200)
 
-  def test_options_write(self):
-    response = requests.get(self.url_options)
-    self.assertEqual(response.status_code, 200)
+  # def test_options_write(self):
```

Comment:
Is it temporary and you will fix it before the merge or just dead code?
If it's just unused you should remove it completely. If someone need it later he can look at the git commit.

Reply:
Sure, i can remove it altogether. I'm not so experienced in the group workflow, so preferred to left it like this for a while rather than deleting it.

Reply:
I will let the veto on @AUTOMATIC1111  but in my opinion it should be remove before the merge.

Reply:
Oh i have no objections to the removal. i just wanted to do it along with coverage. But there was some problems with adding it, so i'll just remove this test and fix some more issues. I've tried it locally https://github.com/mezotaken/stable-diffusion-webui/pull/2 but very unsuccesful. While using API i managed to succesfully run and show result, but due to testing being made through REST API between two independent processes it just measures tests. 
With different approach, when i use `coverage run`  on subprocess with webui instance it cannot generate coverage data file, tho locally it works. But its's wrong anyway 
![image](https://user-images.githubusercontent.com/32306715/202847825-4e104bac-c0f3-489a-8867-1cfa6643cb48.png)
![image](https://user-images.githubusercontent.com/32306715/202847833-c9904a68-01d3-467f-b064-651540df3862.png)
I dont get it, maybe will try again later in different PR

Reply:
Yes no need to stale this PR. Let's add that in another PR. 👍